src/title.tsx

```diff
@@ -60,12 +60,35 @@ export const Title1 = ({children, as = 'h3', id, right, dataAttributes}: TitlePr
 };
 
 export const Title2 = ({children, as = 'h3', id, right, dataAttributes}: TitleProps): React.ReactElement => {
+    const {textPresets} = useTheme();
+    return (
+        <TitleLayout
+            title={
+                <Text
+                    as={as}
+                    id={id}
+                    mobileSize={textPresets.title2.size.mobile}
+                    desktopSize={textPresets.title2.size.desktop}
+                    weight={textPresets.title2.weight}
+                    mobileLineHeight={textPresets.title2.lineHeight.mobile}
+                    desktopLineHeight={textPresets.title2.lineHeight.desktop}
+                    dataAttributes={{'component-name': 'Title2', ...dataAttributes}}
```

Comment:
IMO, this should be moved to the wrapper. The component is not just this text

Reply:
good call. Fixed